.devcontainer.json

```diff
@@ -0,0 +1,22 @@
+{
+  "forwardPorts": [5173, 1317, 26657, 26656],
+  "portsAttributes": {
+      "5173": {
+        "label": "UI",
+        "onAutoForward": "openBrowser"
+      },
+      "1317": {
+        "label": "API",
+        "onAutoForward": "silent"
+      },
+      "26657": {
+          "label": "RPC",
+          "onAutoForward": "silent"
+      },
+      "26656": {
+        "label": "P2P",
+        "onAutoForward": "silent"
+      }
+  },
+  "postStartCommand": "bash -i -c 'nvm install 18.20 && nvm alias default 18.20' && bash ./make_ports_public.sh 5173 1317 26657"
```

Comment:
@rabi-siddique @amessbee I think we need a lot of testing to sure `./make_ports_public.sh 5173 1317 26657` works every time. If there is flakyness then we should address it.